examples/nextjs/src/app/page.tsx

```diff
@@ -102,17 +102,6 @@ export default function Home() {
     refetchOnWindowFocus: false,
   });
 
-  useEffect(() => {
-    if (swapVenuesQuery.data) {
-      setSwapVenues(swapVenuesQuery.data);
-    }
-  }, [swapVenuesQuery.data]);
-  useEffect(() => {
-    if (bridgesQuery.data) {
-      setBridges(bridgesQuery.data);
-    }
-  }, [bridgesQuery.data]);
-
```

Comment:
i think we need to still set the value for swapVenues and bridges, but default them to false

Reply:
otherwise right now none of the checkboxes do anything